addresses spelling corrected from addressess to addresses (#105)
rsksmart/devportal: WindowsInstall.md

# Windows computers instructions

## Requirements

- ***Ruby*** - including all development headers (ruby version can be checked by running `ruby -v`)
- ***RubyGems*** - it is installed with Ruby (which you can check by running `gem -v`)
- ***GCC and Make*** (install MSYS2 which is included with Ruby installer)
- ***Jekyll*** (version can be checked by running `jekyll -v`)

## Installing the requirements

### Ruby
Check the correct version to install [here](https://github.com/rsksmart/rsksmart.github.io/blob/master/Gemfile)<br/>
file: https://github.com/rsksmart/rsksmart.github.io/blob/master/Gemfile

```
ruby "2.6.3"
```

- Installers for windows: [https://rubyinstaller.org/](https://rubyinstaller.org/)

The version 2.6.3 is not the latest, so you need to go here to find it:
- Old versions: [https://rubyinstaller.org/downloads/archives/](https://rubyinstaller.org/downloads/archives/)

 &nbsp;

> ![Ruby install path](https://github.com/rsksmart/rsksmart.github.io/blob/master/assets/img/windowsInstall/windowsInstall-01.png)

Reviewer: This file, and the other screenshots - are stored in `assets/img/windowsInstall/` -  which gets processed and published together with the rest of the site by jekyll.

Since these images are not intended for use in the site, I would recommend telling jekyll to skip them. One way to do this would be in `_config.yml`, under the `exclude` section.

# Windows computers instructions

## Requirements

- ***Ruby*** - including all development headers (ruby version can be checked by running `ruby -v`)
  - [RVM](https://rvm.io/) is recommended to install and switch between multiple Ruby versions.
- ***RubyGems*** - it is installed with Ruby (which you can check by running `gem -v`)
- ***GCC and Make*** (install MSYS2 which is included with Ruby installer)
- ***Jekyll*** (version can be checked by running `jekyll -v`)

## Installing the requirements

### Ruby
Check the correct version to install [here](https://github.com/rsksmart/rsksmart.github.io/blob/master/Gemfile)<br/>
file: https://github.com/rsksmart/rsksmart.github.io/blob/master/Gemfile

```ruby version
ruby "2.6.3"

> ```

Reviewer: This block should be:

```shell
ruby version
ruby "2.6.3"
```

Reviewer 2: ruby "2.6.3" is written is this way at gemfile.
It is not the result of "ruby version", because "ruby version" do not exists, the correct is ruby -v

- Installers for windows: [https://rubyinstaller.org/](https://rubyinstaller.org/)

> The version 2.6.3 is not the last, so you nedd to go here to find it:

Reviewer: ```suggestion
The version 2.6.3 is not the latest, so you need to go here to find it:
```

- Old versions: [https://rubyinstaller.org/downloads/archives/](https://rubyinstaller.org/downloads/archives/)

 &nbsp;

![Ruby install path](https://github.com/rsksmart/rsksmart.github.io/blob/master/assets/img/windowsInstall/windowsInstall-01.png)

> *Do not install in Program files, if you do it, after it not works because it do not find it.*

Reviewer: ```suggestion
*Do not install in Program files, as your system will be unable to find it if you do so.*
```

&nbsp;

![Check MSYS2](https://github.com/rsksmart/rsksmart.github.io/blob/master/assets/img/windowsInstall/windowsInstall-02.png)

*You need to install MSYS2 which is included with Ruby installer.*

&nbsp;

![Run ridk install](https://github.com/rsksmart/rsksmart.github.io/blob/master/assets/img/windowsInstall/windowsInstall-03.png)

*MSYS2 setup will run start automatically, at the end of Ruby installer.*

If it don't start automatically, at the end of Ruby installer, run `ridk install` to setup MSYS2 and development toolchain. 

&nbsp;

At terminal, choose option **1**

![At terminal, choose option **1**](https://github.com/rsksmart/rsksmart.github.io/blob/master/assets/img/windowsInstall/windowsInstall-04.png)

&nbsp;

When you finished, at terminal, you can check if all is ok:

> ```shell

Reviewer: Add another new line to before the code block here, and the other ones too

Reviewer 2: Hummm, I did not understand...

Reviewer: I'll update this directly

ruby -v
gem -v
```

&nbsp;

### Jekyll

At terminal:
```shell
gem install bundler jekyll
gem -v
```

Verify your installation:
```shell
bundler -v
jekyll -v
```

&nbsp;

## Set up

Clone this repository, and run the following commands in its directory:

```shell
bundle install
```

> If you had this error:

Reviewer: ```suggestion
If you get an error similar to this:
```
